spacecmd/src/spacecmd/utils.py

```diff
@@ -58,7 +58,10 @@
 except ImportError:
     import simplejson as json  # python < 2.6
 
-import rpm
+try:
+    import rpm
+except ImportError:
+    from version_utils import rpm
```

Comment:
I would like to get rid of the `pyhon3-rpm` dependency, it's really annoying not being able to `pip install spacecmd`. What's less clear to me is what fallback makes sense.

If the `version_utils` you're thinking of is https://pypi.org/project/version-utils/, then I'm not sure we can just fall back to it. It's `rpm` module has a different API than `python3-rpm` does. At the very least this change requires additional changes.

Furthermore, I'm not sure that all the logic is correct, we'd need to take a closer look. What does the rest of @uyuni-project/python think?

Reply:
From what I remember it only needs the `rpm` package to be able to check version strings to version-utils is an appropriate fallback.